src/app/core/rpc/peer.service.ts

```diff
@@ -51,6 +54,11 @@ export class PeerService {
       error => this.log.er(`updatePeerListLoop(): getpeerinfo error ${error}`)
     );
 
+    this._rpc.call('getblockcount').subscribe(
```

Comment:
Why would you do this?

Reply:
We already have it in state..

Reply:
It caused a hanging in the last few blocks, it would keep the sync modal open longer than required